pkg/k8shandler/clusterloggingrequest.go

```diff
@@ -25,6 +25,14 @@ type ClusterLoggingRequest struct {
 
 	// OutputSecrets are retrieved during validation and used for generation.
 	OutputSecrets map[string]*corev1.Secret
+
+	//CLFVerifier is a collection of functions to control verification
+	//of ClusterLogForwarding
+	CLFVerifier ClusterLogForwarderVerifier
```

Comment:
Not sure this is the best approach. We need to customize the verify behavior per-output, not for the whole forwarder,

Perhaps a map of map[string]VerifyFunc = { "cloudwatch": ...; "kafka": ... }, so Normalize can call the right verify func for each output type.

Reply:
This was added to allow the internal config generator bin to work without being connected to the cluster and really is not intended to be used otherwise.  We can repurpose it if needed but ATM I don't seed the need.

Reply:
OK, understood. However we still need different secret verification for cloudwatch (needs "aws_...") vs. other TLS outputs. We could have a global map of verify functions keyed by output type, and have output types with special needs put a function in the map that the validation code will call if not nil. What do you think?

Reply:
sure.  if that is needed seems reasonable

Reply:
Not needed for this PR. This is OK